### PR TITLE
Update `SymmetricIntersections`

### PR DESCRIPTION
**Pull request by StevellM**
Update of the project "SymmetricIntersections" in the experimental of Oscar.
- Adopt the new suggestions for developers: experimental template, file name, collection of exports, printing conventions;
- Add a "README" about the project;
- Add the contact of @simonbrandhorst (and mine) as maintainer(s) of the project;
- Change name of some functions/objects;
- Remove an unused and untested file (forgotten in the initial commits).

Similarly to the exports, I have collected all the show functions into a separated file to make it easier for possible future changes (easier in the sense of "finding where are the show functions for that particular type")

### CODE REVIEW

File: experimental/SymmetricIntersections/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+### Symmetric Grassmannians
+
+We call here *Symmetric Grassmannians* spaces which parametrize submodules of a
+given group algebra modules. In some sense, we add a group action to the usual
+Grassmannian variety, and look for invariant subspaces of a fixed vector space
+on which a group acts.
+Even though some functionalities about them are available, the naming
+"Symmetric Grassmannians" is not standard and the usage is quite limited. This
+is why the content related to them is not directly exported to the user:
+one may used "Oscar.bla(arguments, of, bla)" to call the corresponding function
```
Comment (simonbrandhorst) [inline experimental/SymmetricIntersections/README.md]:
```suggestion
one may use "Oscar.bla(arguments, of, bla)" to call the corresponding function
```

File: experimental/SymmetricIntersections/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+"bla".
+
+Given a group algebra modules $M$, represented by a linear representation of a
```
Comment (simonbrandhorst) [inline experimental/SymmetricIntersections/README.md]:
```suggestion
Given a group algebra module $M$, represented by a linear representation of a
```

File: experimental/SymmetricIntersections/README.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+finite group, one can create the space parametrising submodules $M$ of a given
+dimension, with given character or with given determinant character. Once
+created, it is possible to ask for a *general element*, some parametrization
```
Comment (simonbrandhorst) [inline experimental/SymmetricIntersections/README.md]: general in which sense?

**Review by simonbrandhorst — COMMENTED** [review]
Can you adapt the show function to the recent  changes in AA, like decapitalization?

File: experimental/SymmetricIntersections/README.md (hunk 3 above)
Comment (StevellM) [reply experimental/SymmetricIntersections/README.md]: Hum I did not chose the best word here, since it is not general.. I will modify this.

**Review by fingolfin — APPROVED** [review]
Looks good to me

@simonbrandhorst please merge if you are OK with it

**Pull request merged**